bundles/org.openhab.binding.automower/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -42,16 +44,10 @@
 		<label>Automower</label>
 		<description>An automatic lawn mower</description>
 
-		<channels>
-			<channel id="name" typeId="nameType"/>
-			<channel id="mode" typeId="modeType"/>
-			<channel id="activity" typeId="activityType"/>
-			<channel id="state" typeId="stateType"/>
-			<channel id="last-update" typeId="lastUpdateType"/>
-			<channel id="battery" typeId="batteryType"/>
-			<channel id="error-code" typeId="errorCodeType"/>
-			<channel id="error-timestamp" typeId="errorTimestampType"/>
-		</channels>
+		<channel-groups>
+			<channel-group id="mower-status" typeId="mowerStatus"/>
+			<channel-group id="mower" typeId="mowerCommands"/>
+		</channel-groups>
```

Comment:
The grouping of the Channels makes the binding incompatible with current configurations. That means existing users need to remove their mower Thing, re-add it and adjust their Items configuration. Although this breaking change is allowed during the OH 2.5.x -> 3.0.0 release, I think it would annoy the users. Is this change really worth it?

Adding new Channels without touching existing ones is okay.

Reply:
Thanks for that comment. I absolutely agree with that. Will update it accordingly